Makefile

```
@@ -159,6 +160,7 @@ build/nginx-drupal: build/nginx images/nginx-drupal/Dockerfile
 build/varnish: build/commons images/varnish/Dockerfile
 build/varnish-drupal: build/varnish images/varnish-drupal/Dockerfile
 build/redis: build/commons images/redis/Dockerfile
+build/redis-persistent: build/commons images/redis-persistent/Dockerfile
```

as `redis-persistent` depends on `redis` we need to reflect that here.
```suggestion
build/redis-persistent: build/redis images/redis-persistent/Dockerfile
```